### ISSUE TITLE
Property 'has' does not exist on type 'CacheInterface'

### ISSUE THREAD

**Issue by bingtsingw** [Kaperskyguru/kap-adonis-cache#16]
`Cache.has()` is available in readme, but seems unavailable

**Comment by bingtsingw** [comment]
I see an pr(https://github.com/Kaperskyguru/adonis-cache/pull/15) has merged, could you please release a new version, thank you

**Comment by shiny** [comment]
It hasn't been included in latest version 2.1.1.

**Comment by shiny** [comment]
Here is a quick fix:
`contracts/cache.ts`

```typescript
declare module '@ioc:Kaperskyguru/Adonis-Cache' {
    interface CacheInterface {
        has(name: string): Promise<boolean>
    }
}
```
